config/imsim-config-instcat.yaml

```diff
@@ -9,369 +9,30 @@
-
-    camera: LsstCam
-
-    exptime: $exptime
-
-    cosmic_ray_rate: 0.2
-
-    det_num:
-        type: Sequence
-        nitems: 189
-        first: 94  # Can set first to something if you want to do a specific sensor.
-
-    dir: fits
-    file_name:
-        type: FormattedStr
-        format : eimage_%08d-%1d-%s-%s-det%03d.fits
-        items:
-            - { type: OpsimData, field: observationId }
-            - { type: OpsimData, field: snap }
-            - { type: OpsimData, field: band }
-            - "$det_name"   # A value stored in the dict by LSST_CCD
-            - "@output.det_num"
-
-    readout:
-        # Convert from e-image to realized amp images
-        readout_time: 3.
-        dark_current: 0.02
-        bias_level: 1000.
-        pcti: 1.e-6
-        scti: 1.e-6
-        bias_levels_file: LSSTCam_bias_levels_run_13421.json
-
-        file_name:
-            type: FormattedStr
-            format : amp_%08d-%1d-%s-%s-det%03d.fits.fz
-            items:
-                - { type: OpsimData, field: observationId }
-                - { type: OpsimData, field: snap }
-                - { type: OpsimData, field: band }
-                - "$det_name"
-                - "@output.det_num"
-
-    truth:
-        dir: output
-        file_name:
-            type: FormattedStr
-            format : centroid_%08d-%1d-%s-%s-det%03d.txt.gz
-            items:
-                - { type: OpsimData, field: observationId }
-                - { type: OpsimData, field: snap }
-                - { type: OpsimData, field: band }
-                - "$det_name"
-                - "@output.det_num"
-        columns:
-            object_id: "@object_id"
-            ra: "$sky_pos.ra.deg"
-            dec: "$sky_pos.dec.deg"
-            x: "$image_pos.x"
-            y: "$image_pos.y"
-            # TODO: ... more probably
-
-gal:
-    type: InstCatObj
+# Get most of the configuration from the imSim config-template
+# for instance catalogs.
+template: imsim-config
 
 ################################################################
 #  Above is template material.
 #
 #  The following entires are added  to the base configuration above.
 ################################################################
 
-# input.instance_catalog:
-#     # This should be overridden below or on the command line.
-#     file_name: default_catalog_file.txt
-#     sed_dir: $os.environ.get('SIMS_SED_LIBRARY_DIR')
+input.instance_catalog:
+    # This should be overridden below or on the command line.
+    file_name: default_catalog_file.txt
+    sed_dir: $os.environ.get('SIMS_SED_LIBRARY_DIR')
 
-# input.opsim_data:
-#     # Read the visit meta data by default we use the same file as the input.
-#     # However you could specify a opsim file or a another instance catalog
-#     # instead.
-#     file_name: '@input.instance_catalog.file_name'
+input.opsim_data:
+    # Read the visit meta data by default we use the same file as the input.
```

Comment:
This should be two sentences.  And rather than "input", this should be something like "above instance catalog with the object information".

Reply:
Done.